src/main/java/ModelsTest/UserTest/UserDaoTest.java

```diff
@@ -0,0 +1,91 @@
+package ModelsTest.UserTest;
+
+import Models.User.DuplicateEmailPresentException;
+import Models.User.User;
+import Models.User.UserDao;
+import Models.User.UserNotFoundException;
+import jdk.nashorn.internal.ir.annotations.Ignore;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.PrintStream;
+
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
I've just noticed I've used a Wildcard import here. Generally, these are inefficient as we don't need to be importing the entire Assertions library, as I'm only using a tiny portion of it.

I've edited my Intellij settings to increase the threshold of imports before auto-importing a wildcard.
![image](https://user-images.githubusercontent.com/35970939/101761645-55afe280-3ad4-11eb-927f-fb69b76f7cb3.png)

I'll push the change in a moment.